tools/greenbone-nvt-sync.in

```diff
@@ -632,6 +635,17 @@ do_feedversion () {
   fi
 }
 
+create_feed_lock_file ()
+{
+  if [ -f $OPENVAS_RUN_DIR/feed-update.lock ] ; then
+    stderr_write "Another process related to the feed update is already running"
```

Comment:
You are using `stderr_write` which would print the message to `stderr` and not necessarily to the log. Is there a reason for not using `log_err` (or `log_warning` for that matter)?